source/mobile/build-custom-mobile-apps.rst

```diff
@@ -0,0 +1,84 @@
+Building and Distributing Your Own Custom Mattermost Mobile Apps
+================================================================
+
+You can build and distribute custom versions of the Mattermost Mobile App. Choosing this approach means `you've decided not to use the mobile app Mattermost has made available <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ through public app stores.
+
+This approach is recommended for:
+
+- Organizations that want to customize their Mattermost Mobile Apps.
+- Customers using self-hosted Mattermost Team Edition or Enterprise Editions that prefer to host their own push proxy server instead of using one of Mattermost’s hosted versions.
+  
+Deployment Options
+------------------
+
+When you decide to build your own Mattermost Mobile apps, you have multiple ways to deploy it: 
+
+- Submitting your app to `an Enterprise App Store <#using-an-enterprise-app-store>`_.
+- Using `an Enterprise Mobile Management (EMM) provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__.
+- Submitting your app to `public app stores <#using-public-app-stores>`_.
```

Comment:
For this one, I believe there are some branding guidelines that they need to follow. see https://handbook.mattermost.com/operations/operations/company-processes/publishing/publishing-guidelines/brand-and-visual-design-guidelines#name-usage-guidelines

I see you added this below, but perhaps makes sense to add it here as well?

Reply:
0/5

Reply:
Given that this content is located on the same page as the link to the branding details in our Handbook, it would be best to limit the number of links to that destination page to a single link.